### PR TITLE
FEATURE: Add markdown to banner description

### PR DESCRIPTION
**Pull request by Grubba27**
you can render markdown in the banner description

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -26,10 +26,22 @@ module ::DiscourseCustomTopicLists
...
+      end
+    raw_custom_lists.map do |list|
+      markdown =
+        Redcarpet::Markdown.new(
+          Redcarpet::Render::HTML.new(hard_wrap: true),
+          no_intra_emphasis: true,
+          fenced_code_blocks: true,
+          autolink: true,
+        )
+      list["description"] = markdown.render(list["description"])
```
Comment (CvX) [inline plugin.rb:43]: There's no need for any external gems. I'm sure what you need is our md rendering pipeline:

```suggestion
      list["description"] = PrettyText.cook(list["description"])
```

Comment (Grubba27) [reply plugin.rb]: Thanks a lot!! I was trying to find a way to use it on core. 


I'll remove references of the `redcarpet`

**Comment by Grubba27** [comment]
In my environment is like this:
Screenshot 2024-06-14 at 15 16 57

Screenshot 2024-06-14 at 15 18 49

Something might be off in your environment, I think.

Could you restart your server?

**Comment by gormus** [comment]
I was 7 commits behind, that's why it didn't work for me i guess, sorry :-D 

I pulled the latest, and also restarted my local; and now it works perfectly.

One suggestion though, could we change `Banner label` to `Banner description`?

File: plugin.rb [hunk 2]
```diff
@@ -26,10 +25,15 @@ module ::DiscourseCustomTopicLists
...
-      current_user.in_any_groups?(allowed_groups)
+    raw_custom_lists =
+      custom_lists.select do |list|
+        allowed_groups = list["access"].split(/(?:,|\s)\s*/).map(&:to_i)
+        allowed_groups = [Group::AUTO_GROUPS[:everyone]] if allowed_groups.empty?
+        current_user.in_any_groups?(allowed_groups)
+      end
+    raw_custom_lists.map do |list|
+      list["description"] = PrettyText.cook(list["description"])
+      list
```
Comment (CvX) [inline plugin.rb:36]: 🏌️ 

```suggestion
    custom_lists.select! do |list|
      allowed_groups = list["access"].split(/(?:,|\s)\s*/).map(&:to_i)
      allowed_groups = [Group::AUTO_GROUPS[:everyone]] if allowed_groups.empty?
      current_user.in_any_groups?(allowed_groups)
    end

    custom_lists.each do |list|
      list["description"] = PrettyText.cook(list["description"])
```

**Review by CvX — APPROVED** [review]

File: plugin.rb (hunk 2 above)
Comment (Grubba27) [reply plugin.rb:36]: I did not know this! thanks for that as well

**Pull request merged**